src/AstTranslator.cpp

```diff
@@ -98,31 +98,21 @@ size_t AstTranslator::getEvaluationArity(const AstAtom* atom) const {
     }
 }
 
-void AstTranslator::translateDirectives(std::map<std::string, std::string>& directives,
-        const AstRelation* rel, const std::string& filePath, const std::string& fileExt) {
+void AstTranslator::translateDirectives(
+        std::map<std::string, std::string>& directives, const AstRelation* rel) {
     // set relation name correctly
     directives["name"] = getRelationName(rel->getQualifiedName());
+
     // set a default IO type of file and a default filename if not supplied
     if (directives.find("IO") == directives.end()) {
         directives["IO"] = "file";
     }
 
-    // load intermediate relations from correct files
-    if (directives.at("IO") == "file") {
-        // set filename by relation if not given
-        if (directives.find("filename") == directives.end()) {
-            directives["filename"] = directives.at("name") + fileExt;
-        }
-    }
     // legacy support for SQLite prior to 2020-03-18
     // convert dbname to filename
     if (directives.at("IO") == "sqlite" && directives.find("dbname") != directives.end()) {
```

Comment:
Is this still required?

Reply:
Anyone using sqlite before 2020-03-18 would be using dbname, so I think it is still required. It could be moved into the sqlite reader/writer, though, same as for filename.